.test-infra/metrics/sync/github/sync_workflows.py

```diff
@@ -134,13 +134,11 @@ def fetchWorkflowData():
                 workflows.append(workflowsPage)
         for pageItem in workflows:
             for item in pageItem:
-                path =item['path']
-                isPostCommit = re.search('(.*)postcommit(.*)',path)
-                if isPostCommit:
-                    result = re.search('/(.*).yml', path)
-                    path =(result.group(1)) + ".yml"
-                    workflowObject = Workflow(item['id'],item['name'],path)
-                    WORKFLOWS_OBJECT_LIST.append(workflowObject)
+                path = item['path']
+                result = re.search('/(.*).yml', path)
+                path = (result.group(1)) + ".yml"
+                workflowObject = Workflow(item['id'],item['name'],path)
+                WORKFLOWS_OBJECT_LIST.append(workflowObject)
             url = "https://api.github.com/repos/apache/beam/actions/workflows/"
             queryOptions = { 'branch' : 'master', 'per_page' : GH_WORKFLOWS_NUMBER_EXECUTIONS,
```

Comment:
our phrase trigger implementation also runs the workflow firstly on master - then checkout the PR inside the workflow, and might be reported as master. Is it able to make the query to event:schedule only?

Reply:
My concern is that some workflows don't have scheduled runs (e.g. the ones around release automation)

Reply:
The if the trigger phrase does not match the workflow, it is skipped; if it matches the workflow runs to end, so I expect it won't get cancelled

I would understand the workflows does not have scheduled run not in the scope of "PostCommit" - and they are handful of them and we can either use different filter (if workflow name does not match PreCommit or PostCommit) or do not include them in postcommit dashboard

Reply:
Ah got it. Agreed on getting this in, but I can follow up to either filter to scheduled runs or just filter out runs with a comment trigger (I wish I could just do an excludes filter, but the API doesn't support it)